oms/src/main/java/com/gfg/oms/Request.java

```diff
@@ -0,0 +1,19 @@
+package com.gfg.oms;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+
+import java.io.Serializable;
+
+@Getter
+public class Request implements Serializable {
+    protected final int api = 1;
+    protected final int id = 1;
```

Comment:
Does anybody know what it is? Is it "correlation id"? Should I set it to some random value for syncronous requests?

Reply:
It is a legacy field, which no one knows why it is needed.

Reply:
Thanks, less things to do. So let both of them be equal `1` always.